packages/opds-web-client/src/stylesheets/book.scss

```diff
@@ -43,28 +59,67 @@
   margin-bottom: $padding;
 
   .compact-info {
-    background-color: rgba($pagecolor, 0.8);
-    border-top: 2px solid $linkcolor;
+    display: flex;
+    flex-direction: column;
+    background-color: rgba($pagecolor, 0.75);
+    border: 1px solid $pagetextcolor;
+    border-top: 2px solid;
+    color: $pagetextcolor;
```

Comment:
I think I would like this better if it was the link color instead of the page text color - and it is a link.

Reply:
I see, but since the dark color was in @ricardoom 's mockup, I'm leaving it as is.

Reply:
It is a link, maybe it could be underlined to better represent that. Thoughts @ricardoom ?